src/content/docs/logs/logs-everywhere/telemetry-in-context.mdx

```diff
@@ -0,0 +1,219 @@
+
+When you update to the supported agent version, no additional installation or configuration is required! Your logs will flow into New Relic with related telemetry data for your apps and hosts.
+
+**REVIEWER: If I didn't know what to look for, how would I know that I'd got telemetry in context in New Relic One?**
+
+Supported agents include:
+
+* Infrastructure monitoring agent [v. CONFIRM or higher](/docs/release-notes/infrastructure-release-notes/infrastructure-agent-release-notes)
+* Java agent [v. CONFIRM or higher](/docs/release-notes/agent-release-notes/java-release-notes) for Logback
+* .NET agent [v. CONFIRM or higher](/docs/release-notes/agent-release-notes/net-release-notes) for Log4Net
+* Ruby agent [v. CONFIRM or higher](/docs/release-notes/agent-release-notes/ruby-release-notes) (default logger)
+
+If your APM agent does not support our telemetry in context solution, you can continue to use our [standard logs in context solutions](/docs/logs/logs-context/logs-in-context).
+
+## Get started [#get-started]
+
+Ready to get started?
+
+1. If you don't have one already, [create a New Relic account](https://newrelic.com/signup). It's free, forever.
+2. Update to the [supported agent version](#agents) for your apps and hosts.
+3. **REVIEWER: Use the guided install for infrastructure in the UI?**
+4. That's it! Now, to start troubleshooting with relevant telemetry data in your logs, go to New Relic One:
+
+* Explorer UI at [one.newrelic.com](https://one.newrelic.com)
+* Explorer UI for EU region data center if applicable: [one.eu.newrelic.com](https://one.eu.newrelic.com)
+
+
+Your logs automatically include attributes such as `span.id`, `trace.id`, `hostname`, `entity_guid`, and more. These make it easier to query and facet on related telemetry data.
+
+You can also add more attributes to make your troubleshooting focus even more precise. For example, you may want to add more attributes to get other key telemetry data specific to your ecosystem. 
+
+**REVIEWER: Can you give a few examples of the types of attributes I might want to add? Is this referring to [infrastructure log's optional configuration documentation](https://docs.newrelic.com/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent#optional-config), or are there other things they might want to add?**
+
+## Avoid ingest limits and extra billing [#ingest]
+
+Using telemetry in context functionality will increase your data ingest, and this may have an impact on your ingest limits and billing. For example, when your engineering team is troubleshooting a problem with your app, it may be helpful to temporarily set the limits to maximum data collection. (By default, the maximum number of logs sampled is 10,000. **REVIEWER: Is 10,000 max correct?**) But if you leave this running for several days, this could result in a lot of unnecessary data and wasted money.
```

Comment:
Yes, 10,000 is correct! (Though technically, for the Ruby Agent the closest divisor is 9,996)